### ISSUE TITLE
[BUG] High pressure steam machines have no speed benefits

### ISSUE THREAD

**Issue by Akiraca** [GregTechCE/GregTech#1324]
**Describe the bug**
Re-raise of the unsolved issue #332. I tested with macerators and the OP tested with furnances.

**Versions**
Forge: 14.23.5.2854
GTCE: 1.10.5.582
_Addons:_ Gregicality

**Setup**
Playing Solo

**Steps To Reproduce**
1) Put down a steam macerator and a high pressure steam macerator.
2) Supply them with steam.
3) Put in ores for pulverization.
4) No speed difference observed.

**Expected behavior**
High pressure steam machines being faster than regular ones.

**Comment by galyfray** [comment]
This is not a bug hight pressure machines are overclocked low pressure ones. This means that when you use recipes that are close to the lv voltage you won't see any clear difference try with a forge hammer with a 8Eu/t recipe and you should see a speed increase . However I agree whith the fact that hight pressure machines should allways beeing faster than low pressure ones.

**Comment by LAGIdiot** [comment]
Thank you for bring in it up. I can confirm that high pressure steam machines are quite under buffed but I don't consider it bug. As already mentioned they can overclock recipe (but I am not sure how many of them will trigger on this). Of course we could look in buffing them in some meaningful way.

**Comment by galyfray** [comment]
maybe buffing the overcloking behavior of the steam machines could help ? If I remember well multiplying the EU/t per 4 only double the speed of the recipes maybe we can switch to a 4 to 3 ratio instead of the actual 4 to 2 ? (I fear that 4 to 4 will make high pressure machines faster than LV machines).

**Comment by LAGIdiot** [comment]
No, touching overclocking is suicide as it is fragile, unstable and intertwined in everything recipe related.

**Comment by leagris** [comment]
Also High-pressure steam machines are able to process MV+ recipes
I could process darkwood, ghostwood in the HP extractor but the Recipe is 120 EU/t

**Comment by galyfray** [comment]
> Also High-pressure steam machines are able to process MV+ recipes
> I could process darkwood, ghostwood in the HP extractor but the Recipe is 120 EU/t

mm strange steam machines are noted as Lv machines in terms of tier. This kind of things should not append.



> No, touching overclocking is suicide as it is fragile, unstable and intertwined in everything recipe related.

Really ? I mean overloading the calculate overclock doesn't seems to be this dangerous.

**Comment by Akiraca** [comment]
>This means that when you use recipes that are close to the lv voltage you won't see any clear difference try with a forge hammer with a 8Eu/t recipe and you should see a speed increase .

Most ore pulverization recipes have 12Eu/t. Should they be overclocked or not?

**Comment by galyfray** [comment]
In theory they should get overclocked in practice due to the strange implementation of the overcloking it does not. I don't know what implementation choice lead to this but I feal like this way isn't the right way to overclok as it is hard top understand and in practice complicated to predict.
